File: RQNOJ/12/main.cpp
#include <cstdio>
int a[10];
int h,ans;
int main(){
	for (int i=0;i<10;++i) scanf("%d",a+i);
	scanf("%d",&h);
	for (int i=0;i<10;++i) ans+=(a[i]<=h+30)?1:0;
	printf("%d\n",ans);
	return 0;
}

File: RQNOJ/12/main.java
import java.util.*;
class main{
	public static void main(String args[]){
		int a[]=new int[10];
		int w=0;
		Scanner cin=new Scanner(System.in);
		for (int i=0;i<10;++i){
			a[i]=cin.nextInt();
		}
		int ans=0;
		w=cin.nextInt()+30;
		for (int i=0;i<10;++i){
			if (w>=a[i]) ans++;
		}
		System.out.println(ans);
	}
}
